avocado/utils/disk.py

```diff
@@ -9,18 +9,84 @@
 #
 # See LICENSE for more details.
 #
-# This code was inspired in the autotest project,
+# Copyright: 2017 IBM
+# Author: Narasimhan V sim@linux.vnet.ibm.com
 #
+# freespace() function was inspired in the autotest project,
 # client/base_utils.py
 
 
 """
 Disk utilities
 """
 
+import logging
 import os
+from . import process
+
+
+LOGGER = logging.getLogger('avocado.test')
+
+
+class FSException(Exception):
+    """
+    Base Exception Class for all exceptions
```

Comment:
Yep, I agree that the docstrings need to be corrected. Right now the code has a copy/paste mistake.